history/characters/56000_mossflayer.txt

```diff
@@ -68,3 +68,20 @@
 	561.6.12={ birth=yes trait=creature_troll }
 	612.1.24={ death=yes }
 }
+56008={
```

Comment:
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Coder-Guidelines#history-guidelines
Due to our guidelines.
```suggestion
zulrogg={
```